integration-cli/docker_api_images_test.go

```diff
@@ -128,3 +128,38 @@ func (s *DockerSuite) TestAPIImagesSearchJSONContentType(c *check.C) {
 	c.Assert(res.StatusCode, checker.Equals, http.StatusOK)
 	c.Assert(res.Header.Get("Content-Type"), checker.Equals, "application/json")
 }
+
+// Test case for 30027
```

Comment:
nit: please add a brief description about #30027? (e.g. "Image size reported as -1 in v1.12 client against v1.13 daemon")